examples/quarkus-jib-extension/pom.xml

```
@@ -0,0 +1,125 @@
+<?xml version="1.0"?>
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd" xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>org.acme</groupId>
+  <artifactId>getting-started</artifactId>
```

Following the format of the other examples in the repo: 

```suggestion
  <groupId>example</groupId>
  <artifactId>quarkus-jib-extension</artifactId>
```